updating/updating-hardware-on-nodes-running-on-vsphere.adoc

```diff
@@ -10,11 +10,21 @@ You must ensure that your nodes running in vSphere are running on the hardware v
 
 You can update your virtual hardware immediately or schedule an update in vCenter.
 
+[IMPORTANT]
+====
+Before upgrading OpenShift 4.12 to OpenShift 4.13, you must update vSphere to *v7.0.2 or later*; otherwise, the OpenShift 4.12 cluster will be marked *un-upgradeable*.
+====
+
 [id="updating-virtual-hardware-on-vsphere_{context}"]
 == Updating virtual hardware on vSphere
 
 To update the hardware of your virtual machines (VMs) on VMware vSphere, update your virtual machines separately to reduce the risk of downtime for your cluster.
 
+[IMPORTANT]
+====
+As of OpenShift 4.12, VMware virtual hardware version 13 is no longer supported. You need to update to VMware version 15 or later for supporting functionality.
```

Comment:
s/Openshift/{product-title}

Reply:
I'm not current on vSphere, but it seems strange to have "As of OpenShift 4.12" statements in a pull request claiming:

> Version(s):
> 4.13+

If HW 13 support is removed in 4.12, wouldn't the 4.11 docs grow a deprecation notice, and the 4.12 release notes include a removal notice?  In fact, the 4.12 release notes [currently say][1]:

> These versions are still fully supported, but version 4.12 of OpenShift Container Platform requires vSphere virtual hardware version 15 or later. For more information, see [Deprecated and removed features](https://docs.openshift.com/container-platform/4.12/release_notes/ocp-4-12-release-notes.html#ocp-4-12-deprecated-removed-features).

And [also][2] pointing out that HW 13 support was removed in 4.11!  So I'm not clear if HW 13 support was removed in 4.11 or 4.12, but either way, I don't see why we need to talk about it in 4.13 docs, even in the context of 4.12-to-4.13 updates, because it was already unsupported on 4.12.

[1]: https://docs.openshift.com/container-platform/4.12/release_notes/ocp-4-12-release-notes.html#ocp-4-12-csi-driver-install-vsphere
[2]: https://docs.openshift.com/container-platform/4.12/release_notes/ocp-4-12-release-notes.html#ocp-4-12-removed-features-virt-hw-v13

Reply:
Misspoke - it's as of 4.13 :) updated